Toggl.Droid/Autocomplete/AutocompleteExtensions.cs

```diff
@@ -19,6 +19,17 @@ public static class AutocompleteExtensions
         public static IImmutableList<ISpan> AsImmutableSpans(this ICharSequence text, int cursorPosition)
             => text.AsSpans(cursorPosition).ToImmutableList();
 
+        public static bool isEqualTo(this ISpannable first, ISpannable second)
+        {
+            if (first.ToString() != second.ToString()) return false;
+
+            var firstSpans = first.AsImmutableSpans(0);
+            var secondSpans = second.AsImmutableSpans(0);
+
+            if (firstSpans.Count != secondSpans.Count) return false;
+            return firstSpans.Zip(secondSpans, (f, s) => f.Equals(s)).All((isEqual) => isEqual);
```

Comment:
```suggestion
            return firstSpans.Zip(secondSpans, (f, s) => f.Equals(s)).All(isEqual => isEqual);
```

The parentheses are not needed. Even better, maybe use:

```cs
            return firstSpans.Zip(secondSpans, (f, s) => f.Equals(s)).All(CommonFunctions.Identity);
```

Also, please format linq operators each in a new line (except maybe the first one).

Reply:
`CommonFunctions.Identity` looks cool but seems less readable to non-C# guy like me so I would still prefer the original solution if there is no big argument against it.

Reply:
Just found out `CommonFunctions.Identity` is not some C# `deep knowledge` but actually part of our codebase that we use normally.  I guess I will use it after all.